instructure/canvas-lms
# Added live events support for attachments + wiki pages fix

This PR brings in support for creating, updating and removing attachments.

It also fixes an issue when deleting a wiki page wouldn't trigger a live event

Test plan:
  * create attachments by uploading course files
  * update display name of course files
  * delete some course files
  * delete a course wiki page
  * ensure appropriate messages are in the kinesis queue
